src/pages/pt-BR/guides/markdown-content.md

```diff
@@ -0,0 +1,439 @@
+
+## Páginas Markdown
+
+Astro trata qualquer arquivo `.md` dentro do diretório `/src/pages` como uma página. Colocar um arquivo nesse diretório ou em qualquer subdiretório criará, automaticamente, uma rota de página usando o nome do caminho do arquivo.
+
+📚 Leia mais sobre o [roteamento baseado em arquivo](/pt-BR/core-concepts/routing) do Astro.
+
+### Exemplo Básico
+
+A maneira mais fácil de começar a usar Markdown no Astro é criar uma rota de página inicial `src/pages/index.md` em seu projeto. Copie o template básico abaixo em seu projeto e visualize o HTML renderizado na rota da página inicial do seu projeto. Geralmente, em [http://localhost:3000](http://localhost:3000/).
+
+```markdown
+---
+# Exemplo: src/pages/index.md
+title: Olá, mundo!
+---
+
+# Olá!
+
+Esta é sua primeira página em markdown. Provavelmente, ela não tem muito estilo, embora o Markdown ofereça suporte para **negrito** e _itálico_.
+
+Para saber mais sobre como adicionar um layout à sua página, leia a próxima seção **Markdown Layouts**.
+```
+
+### Markdown Layouts
+
+Páginas Markdown têm uma propriedade especial frontmatter para `layout` que define o caminho relativo para um [componente de layout](/pt-BR/core-concepts/layouts) Astro. Este componente envolverá seu conteúdo Markdown, fornecendo uma casca de página e quaisquer outros elementos de template de página incluídos.
+
+```markdown
+---
+layout: ../layouts/BaseLayout.astro
+---
+```
+
+Um layout típico para páginas Markdown inclui:
+
+1. a prop `content` para acessar os dados do frontmatter da página Markdown.
+2. um [`<slot />`](/pt-BR/core-concepts/astro-components#slots) default para indicar onde o conteúdo Markdown da página deve ser renderizado.
+
+```astro
+---
+// src/layouts/BaseLayout.astro
+// 1. A prop content dá acesso aos dados do frontmatter
+const { content } = Astro.props;
+---
+<html>
+  <head>
+    <!-- Adicione outros elementos Head aqui, como estilos e meta tags. -->
+    <title>{content.title}</title>
+  </head>
+  <body>
+    <!-- Adicione outros componentes de UI aqui, como cabeçalhos e rodapés comuns. -->
+    <h1>{content.title} por {content.author}</h1>
+    <!-- 2. O HTML renderizado será passado para o slot default. -->
+    <slot />
+    <p>Escrito em: {content.date}</p>
+  </body>
+</html>
+```
+
+A prop `content` também contém uma propriedade `astro` com metadados adicionais sobre a página, como o objeto Markdown `source` completo e um objeto `headers`.
+
+Um exemplo de objeto `content` de post de blog pode ser algo como:
+
+```json
+{
+  /** Frontmatter from a blog post
+  "title": "Lançamento do Astro 0.18",
+  "date": "Terça-feira, 27 de julho de 2021",
+  "author": "Matthew Phillips",
+  "description": "Astro 0.18 é o nosso maior lançamento desde o lançamento do Astro.",
+  "draft": false,
+  "keywords": ["astro", "lancamento", "anuncio"]
```

Comment:
Adding the missing translations here. `draft` isn't translated because it makes part of Astro itself.
```suggestion
  /** Frontmatter de uma postagem de blog
  "titulo": "Lançamento do Astro 0.18",
  "data": "Terça-feira, 27 de julho de 2021",
  "autor": "Matthew Phillips",
  "descricao": "Astro 0.18 é o nosso maior lançamento desde o lançamento do Astro.",
  "draft": false,
  "palavras-chave": ["astro", "lancamento", "anuncio"]
```